test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenManagementSpecs.java

```diff
@@ -154,7 +154,6 @@ public HapiApiSpec kycMgmtFailureCasesWork() {
 								.name(salted("name"))
```

Comment:
You can safely remove the `.name`. Its a leftover from before. The name if omitted is automatically set in the `tokenCreate` method